optaplanner-docs/src/modules/ROOT/pages/release-notes/release-notes-8.adoc

```diff
@@ -1,6 +1,24 @@
 [[releaseNotes-8.x]]
 == OptaPlanner 8.x Release Notes
 
+[[releaseNotes-8.31.0.Final]]
+=== OptaPlanner 8.31.0.Final
```

Comment:
Do you want to wait with merging to the next week?

Reply:
Yes. :-)